docs/docs/models-definition.md

```diff
@@ -85,6 +85,7 @@ Sequelize.STRING                      // VARCHAR(255)
 Sequelize.STRING(1234)                // VARCHAR(1234)
 Sequelize.STRING.BINARY               // VARCHAR BINARY
 Sequelize.TEXT                        // TEXT
+Sequelize.TEXT('length')              // TEXT
```

Comment:
I think it would be more readably if you give an example of what `'length'` could be ('tiny', 'medium', and 'long')